expr/generator.go

```diff
@@ -111,8 +111,11 @@ func (a *AggExpr) Eval(rec *zng.Record) (zng.Value, error) {
 	a.src.Init(rec)
 	for {
 		zv, err := a.src.Next()
+		if err == zng.ErrMissing {
+			continue
+		}
 		if err != nil {
```

Comment:
Our usual pattern is to test for `err != nil` first and then, inside the associated if block, test for specific errors.
```suggestion
		if err != nil {
			if err == zng.ErrMissing {
				continue
			}
```